build-system/tasks/release/index.js

```diff
@@ -207,14 +201,20 @@ async function compileDistFlavors_(distFlavors, tempDir) {
           path.join(flavorTempDistDir, 'dist', path.basename(staticFilePath))
         )
       ),
-      // Individual files to copy from the resulting build artifacts.
-      ...Object.entries(POST_BUILD_MOVES).map(([from, to]) =>
-        fs.copy(
-          path.join(flavorTempDistDir, from),
-          path.join(flavorTempDistDir, to)
-        )
-      ),
     ]);
+    const postBuildMovesPromise = !argv.esm
+      ? Promise.all([
+          // Individual files to copy from the resulting build artifacts.
+          // This is only relevant for nomodule builds.
+          ...Object.entries(POST_BUILD_MOVES).map(([from, to]) =>
+            fs.copy(
+              path.join(flavorTempDistDir, from),
+              path.join(flavorTempDistDir, to)
+            )
+          ),
+        ])
+      : Promise.resolve();
+    await Promise.all([...staticFilesPromise, ...postBuildMovesPromise]);
```

Comment:
Since `Promise.all` returns a single promise, I don't think you need to use the rest operator on the two promises.

Reply:
Good catch! Done.